src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -866,7 +866,9 @@ export class MatrixRTCSession extends TypedEventEmitter<MatrixRTCSessionEvent, M
                 this.room.roomId,
                 EventType.GroupCallMemberPrefix,
                 newContent,
-                this.useLegacyMemberEvents ? localUserId : `${localUserId}_${localDeviceId}`,
+                this.useLegacyMemberEvents
```

Comment:
this really should use the computed `legacy` prop instead of the config param.

computed here:
```
const legacy = "memberships" in content || this.useLegacyMemberEvents;
```

Reply:
D'oh, of course, thanks.